### ISSUE TITLE
F541 does not find violations in concatenations

### ISSUE THREAD

**Issue by samueljsb** [astral-sh/ruff#10885]
`ruff` does not correctly identify f-strings with no placeholders if they are being concatenated with other f-strings that *do* have placeholders. e.g:

```python
placeholder = "placeholder"
(
    f"first line no placeholder"
    f"second line with {placeholder}"
)
(
    f"first line with {placeholder}"
    f"second line with  no placeholder"
)
```

```console
$ ruff check t.py
All checks passed!
```

The error is correctly found if there is no other f-string:

```python
placeholder = "placeholder"
(
    f"first line no placeholder"
    "second line with no placeholder"
)
```

```console
$ ruff check t.py
t.py:3:5: F541 [*] f-string without any placeholders
Found 1 error.
[*] 1 fixable with the `--fix` option.
```

This was demonstrated in a clean venv with `ruff` 0.3.5.

**Comment by samueljsb** [comment]
I've opened #10886 to demonstrate this in a test.

**Comment by dhruvmanila** [comment]
Thanks! I think it was not possible to detect this reliably before 017e82911 but it should be trivial now.

**Comment by samueljsb** [comment]
It was agreed in #10886 that this rule, to match the behaviour of `pyflakes`, should not detect this as a violation.

I'm not convinced that there's any deliberate reason for doing that in `pyflakes` (it was brought up in PyCQA/pyflakes#552 but the only discussion was about why the AST is that way rather than why the rule doesn't consider it a problem). But, for compatibility, leaving this rule alone makes sense.

I'll close this issue, since F541 is behaving as expected.
